INSTALL.rst

```diff
@@ -0,0 +1,171 @@
+###########
+Cyclus Installation Guide
+###########
+
+Cyclus uses the CMake cross platform bailed-generator as a build system. TO
+simplify the usage of cmake to build Cyclus from source, a python script is
+provided with the source.
+
+.. contents:: Table of Contents
+
+******************************
+Dependencies
+******************************
+
+In order to facilitate future compatibility with multiple platforms,
+Cyclus is built using `CMake`_. A full list of the Cyclus package
+dependencies is shown below:
+
+====================   ==================
+Package                Minimum Version
+====================   ==================
+`CMake`                2.8
+`boost`                1.46.1
+`libxml2`              2
+`libxml++`             2.36
+`python`               2.7 or 3.3+
+`sqlite3`              3.7.10
+`HDF5`                 1.8.4
+`Coin-Cbc`             2.5
+====================   ==================
+
+You can find `here <DEPENDENCIES.rst>`_  help to install those dependencies for
+the major supported system.
+
+******************************
+Installation
+******************************
+
+Before going further on the installation procedure be sure you have installed
+all the required dependencies listed above. You can also find `there
+<DEPENDENCIES.rst>`_ instruction to install those dependencies depending of the
+system you are working on.
+
+Default installation
+------------------------
+
+Run the install script:
+
+.. code-block:: bash
+  
+  python install.py
+
+
+If you successfully followed the instruction above cyclus binary have been
+generated and be placed in the  ``.local/`` in your home directory. 
+You need to had ``~/.local/bin`` to the bottom of your ``$PATH``:
+
+.. code-block:: bash
+  
+  echo 'export PATH="$HOME/.local/bin:$PATH' >> .bashrc
+
+Custom Cyclus installation
+---------------------------
+The installation using the install script can be customized using the following
+flag:
+
++--------------------------------------------+--------------------------------------------------------------------------------------------------+
```

Comment:
Please use the list-table directive when tables become this large. http://docutils.sourceforge.net/docs/ref/rst/directives.html#list-table